powerpacks.go

```diff
@@ -10,22 +10,36 @@ type PowerpackService struct {
 	Powerpack
 }
 
+type Priority struct {
+	Priority1 *string `json:"priority1"`
```

Comment:
why Prioritys are initialised with pointer string? Are you allowing  empty string ?

Reply:
yes, they could be empty (partial, example: only `"priority2": "tollfree"` is given) for create/update request